api/src/main/java/marquez/jobs/DbRetentionConfig.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018-2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.jobs;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/** Configuration for {@link DbRetentionJob}. */
+public final class DbRetentionConfig {
+  public static final int DEFAULT_FREQUENCY_MINS = 15;
+  public static final int DEFAULT_RETENTION_DAYS = 7;
```

Comment:
3rd constant for the default 😄

Reply:
There's now only a single reference: [`DbRetention.DEFAULT_RETENTION_DAYS`](https://github.com/MarquezProject/marquez/blob/feature/db-retention/api/src/main/java/marquez/db/DbRetention.java#L23)